src/zoid/buttons/util.js

```diff
@@ -70,7 +102,7 @@ export function createVenmoExperiment() : Experiment | void {
 
 export function getVenmoExperiment(experiment : ?Experiment) : VenmoExperiment {
     const enableFunding = getEnableFunding();
-    const isEnableFundingVenmo = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1;
+    const isEnableFundingVenmo = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1 && isDevice();
```

Comment:
One idea is to leverage `supportsQRPay()` so it's more clear on what this `isDevice()` check does. Just thinking about the future if you need to change out what browsers support QRPay then you only have to change that logic in one place `supportsQRPay()`.
```suggestion
    const isEnableFundingVenmo = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1 && supportsQRPay(FUNDING.VENMO);
```